lang/python/python3/Makefile

```diff
@@ -311,6 +311,26 @@ define Host/Install
   endif
 endef
 
+define Host/Uninstall
+	rm -f $(HOST_PYTHON3_DIR)/bin/python{3,$(PYTHON3_VERSION)}
```

Comment:
could we somehow construct the list of files to remove from `Host/Install`?
that way, we would not need to sync it when things change

Reply:
Core has this functionality for `Build/InstallDev` (files installed to a temp directory, make a file list, remove files from file list during package clean) but not `Host/Install` for some reason... I think it would be better to add it to core.

I was hoping I could get away with just this :laughing:

Reply:
I'll try to put together a patch for core today.

Reply:
shall we limit this patch to just ensurepip=upgrade ?
and re-visit this later ?

thing is (for me particularly) i don't really expect much of OpenWrt's build-system to be able to keep track of changes/cleanup;
what i'm saying here is: we will typically recommend to people to clean their build-dir (for a while anyway) if things look weird;

i am noticing that things are getting better though;

people will always try weird things anyway and get weird results, which end up in support questions; no matter how good the system becomes;

Reply:
Sure :joy: